source/tutorials/module-system/module-system.md

```diff
@@ -0,0 +1,1289 @@
+# Deep dive demo: Wrapping the world in modules
```

Comment:
This is too long to fit nicely in the sidebar (see deployed preview)

```suggestion
# Module system
```

Reply:
I wouldn't want this to pretend to be the definitive module system tutorial. The title should also say what something about what to expect. (Also we can change names later on)

Reply:
I mean, it's the *only* module system tutorial? I think I don't understand the argument.

Reply:
```suggestion
# Module system deep dive

Or: *Wrapping the world in modules*
```